### PR TITLE
Fix: Release 2.8.0 grouped issues fix - Summary, Concept pages

### PR DESCRIPTION
**Pull request by Bilelkihal**
## Fixed issues:

- https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/609
Capture d’écran du 2024-05-02 10-25-05

- https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/616

- Fix the style of add comment and add proposal buttons:

image

- https://github.com/agroportal/project-management/issues/489

https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/82af91a2-bc11-43d5-b86f-8e927a964396

- Add the selected bubble to the mappings legend tooltip
image

- Dix concept details raw data values alignment to use divs breaking line (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/94f15a2228f566086760ffd3bb59c2cb24ee665c)  (fix https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/527)

image

- Fix recommender results ontology link to redirect to UI, not API (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/afe29b8c2e6f8a39bebb70fb713f053234891916) (source: https://github.com/agroportal/project-management/issues/508)
image

- Dix summary ofaire tooltip breaking to a new line the card header (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/9d7e6d0025822783df4270f8a27f4ebb48be03ab) (source: https://github.com/agroportal/project-management/issues/508) 
image

- Fix opening the concept tree view link in a new tab showing only HTML (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/9df168a4d467360a63a2077a6d046d1bd05731b9) (fix https://github.com/agroportal/project-management/issues/486) 

- Fix French dates internationalization (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/bc69f237cff28267b8caaf4e9749d6c6c2cf4ef1) (fix https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/604)

- Fix edit submission up front help message in English version (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/0b53f9274b43815fa47677094f89565587a474c1) (fix https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/603)

- Add no properties alert message to the ontology viewer (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/b66b19e5296306e5571d256f39c00a292d832fde) (fix https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/600)

- Localize the mappings visualization legend text (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/88714338ffced784bcdc1d1ab95f3689d7f41e9a)

### CODE REVIEW

File: app/helpers/multi_languages_helper.rb [hunk 1]
```diff
@@ -99,8 +99,10 @@ def content_language_selector(id: 'content_language', name: 'content_language')
 
 
   def language_hash(concept_label)
-
-    return concept_label.first if concept_label.is_a?(Array)
+    if concept_label.is_a?(Array)
```
Comment (syphax-bouazzouni) [inline app/helpers/multi_languages_helper.rb:102]: this is not correct for `prefLabels`, which needs to return only one even there is multiple prefLabels

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/helpers/multi_languages_helper.rb (hunk 1 above)
Comment (Bilelkihal) [reply app/helpers/multi_languages_helper.rb:102]: Is this related only to prefLabels only?

Comment (syphax-bouazzouni) [reply app/helpers/multi_languages_helper.rb:102]: not really, this function `language_hash`has for goal to get you the correct value for a correspondent language. is you choose `all` it returns an hash of the values in all the languages

Comment (Bilelkihal) [reply app/helpers/multi_languages_helper.rb:102]: It returns always a single value !

**Comment by jonquet** [comment]
Parfait. tres bien avec les bloc colorés qui distingue les synonymes.

File: app/helpers/multi_languages_helper.rb (hunk 1 above)
Comment (syphax-bouazzouni) [reply app/helpers/multi_languages_helper.rb:102]: Try to show all languages with AGROVOC.

Comment (syphax-bouazzouni) [reply app/helpers/multi_languages_helper.rb:102]: image

**Comment by syphax-bouazzouni** [comment]
@Bilelkihal for each fixed issue add a screenshot, so I don't need to check it by running the code.

File: app/helpers/mappings_helper.rb [hunk 2]
```diff
@@ -179,7 +179,18 @@ def client_filled_modal
   def mappings_bubble_view_legend
     content_tag(:div, class: 'mappings-bubble-view-legend') do
       mappings_legend_section('Bubble size:', 'The global number of mappings with all other ontologies.', 'mappings-bubble-size-legend') +
```
Comment (syphax-bouazzouni) [inline app/helpers/mappings_helper.rb:181]: not internationalized

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
@Bilelkihal you need localize the mappings legends

File: app/helpers/mappings_helper.rb (hunk 2 above)
Comment (syphax-bouazzouni) [reply app/helpers/mappings_helper.rb]: Done by me in [8871433](https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/88714338ffced784bcdc1d1ab95f3689d7f41e9a)

**Pull request merged**